### PR TITLE
15 delete contact UI

### LINKED ISSUE #15: Create Delete UI for Contact
Ability to delete contact with contact alert

### PR DESCRIPTION
**Pull request by sritabh**
fixes #15

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Notes/Pane/Create.jsx [hunk 1]
```diff
@@ -7,10 +7,10 @@ import Form from "./Form";
...
 const Create = ({ showPane, setShowPane, setNotes }) => {
-  const onClose = () => setShowPane(false);
+  const handleClose = () => setShowPane(false);
 
   return (
-    <Pane isOpen={showPane} onClose={onClose}>
+    <Pane isOpen={showPane} onClose={handleClose}>
       <Pane.Header>
         <Typography style="h2" weight="semibold">
           Create a new note
```
Comment (navaneethsdk) [inline app/javascript/src/components/Dashboard/Notes/Pane/Create.jsx:16]: Translations

File: app/javascript/src/components/Dashboard/Notes/Pane/Edit.jsx [hunk 2]
```diff
@@ -5,16 +5,16 @@ import { Pane, Typography } from "neetoui";
...
 const Edit = ({ showPane, setShowPane, note, setNotes }) => {
-  const onClose = () => setShowPane(false);
+  const handleClose = () => setShowPane(false);
 
   return (
-    <Pane isOpen={showPane} onClose={onClose}>
+    <Pane isOpen={showPane} onClose={handleClose}>
       <Pane.Header>
         <Typography style="h2" weight="semibold">
           Edit note
```
Comment (navaneethsdk) [inline app/javascript/src/components/Dashboard/Notes/Pane/Edit.jsx:14]: Translations

File: app/javascript/src/components/Dashboard/Notes/constants.js [hunk 3]
```diff
@@ -18,21 +18,23 @@ export const NOTES_FORM_VALIDATION_SCHEMA = yup.object().shape({
...
+      label: yup
+        .string()
+        .trim()
+        .required(t("required.entity", { entity: "Assigned Contact" })),
+      value: yup
+        .string()
+        .trim()
+        .required(t("required.entity", { entity: "Assigned Contact" })),
+    }),
   tags: yup.array().min(1, t("required.entities", { entities: "Tags" })),
```
Comment (navaneethsdk) [inline app/javascript/src/components/Dashboard/Notes/constants.js:32]: Shouldn't these be required as well?

**Review by navaneethsdk — COMMENTED** [review]
@sritabh A few comments

**Comment by sritabh** [comment]
@navaneethsdk 
Kindly, look from the latest PR #18 
I have  already made those changes.

Thanks.

File: app/javascript/src/components/Dashboard/Notes/constants.js (hunk 3 above)
Comment (sritabh) [reply app/javascript/src/components/Dashboard/Notes/constants.js:32]: Yes, I have updated that here #18 
I misunderstood when you said to make it `nullable` later I understood what you meant.

**Pull request merged**